channels/hu.m3u

```diff
@@ -53,3 +59,5 @@ http://188.35.9.5:10007/stolen/246r/parasite.install
 https://cloudfront63.lexanetwork.com:1440/hlsrelay014/hls/livestream.sdp.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/xiSeTRL.jpg" group-title="",Ózdi Városi TV
 https://stream.unrealhosting.hu:443/hls/ozdtv/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JCHEuWb.jpg" group-title="",VTV Füzesabony
+https://stream.unrealhosting.hu:443/hls/ftv/live.m3u8
```

Comment:
at start-up gives an error "Unsecured connection"

Reply:
For me this link works on VLC , but normally when you open the link you get a message to accept a certificate , and you need to accept it , if you do the link should play.